app/controllers/missions_controller.rb

```diff
@@ -80,12 +71,10 @@ def order_deliverables
 
   private
 
-  # Use callbacks to share common setup or constraints between actions.
   def set_mission
     @mission = Mission.find(params[:id])
```

Comment:
Might want to start adding some eager loading here, eg: `authorize!` will load the mission's users.